### DISCUSSION TITLE
显存泄露

### DISCUSSION THREAD

**Discussion by speakstone** [snakers4/silero-vad#213] [Q&A] [answered]
在使用本项目时，对于特别长的音频，如50min以上，model(chunk, sampling_rate)执行完后，对于下一个chunk，并不会主动释放显存，导致每个speech_prob推理需要的显存不断增加，这对于大的语音数据将直接显存溢出或cpu溢出

**Comment by snakers4** [comment, marked as answer]
This typically happens when you do not disable gradients in pytorch
